src/app/commands/init_binance_websocket.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from app.clients.binance.binance_client import BinanceClient
+from app.services import DatabaseService
+from app.services.consumer_service import ConsumerService
+from app.services.exchanges.binance_exchange_service import BinanceExchangeService
+from app.services.exchanges.binance_websocket_service import BinanceWebsocketService
+from sqlalchemy.orm import Session
+
+
+def show_help() -> str:
+    return """
+    Start Binance websocket consumption.
+
+    Usage:
+    python manage.py init_binance_websocket PAIR1 PAIR2 ...
+"""
+
+
+def execute(arguments: list) -> None:
+    pairs = arguments[1:]
+
+    logging.basicConfig(level=logging.INFO, format="%(asctime)s %(levelname)s %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
+
+    logging.info("Starting Binance for:")
+    logging.info(", ".join(pairs))
+
+    session = DatabaseService.create_session()
+    populate_binance_data(session)
+
+    # populate_candlesticks(session, pairs)
```

Comment:
Is there any reason to keep this function commented? I noticed that the `consumer_service::populate_candlesticks` has some work to do(feature-5), but maybe a `TODO` should be flagged here instead of just keeping the line commented, what do you think?